src/components/settings/SettingsAccessTab/index.jsx

```diff
@@ -98,7 +98,7 @@ const SettingsAccessTab = ({
               </div>
             )}
 
-            {identityProvider && (
+            {(
```

Comment:
Is this left over from trying to get it to show in the UI temporarily?

Reply:
yep